render/detailed/connections.go

```diff
@@ -115,7 +110,7 @@ func incomingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 		TopologyID:  topologyID,
 		Label:       "Inbound",
 		Columns:     columnHeaders,
-		Connections: connectionRows(r, counts, isInternetNode(n)),
+		Connections: connectionRows(r, ns, counts, isInternetNode(n)),
```

Comment:
I am assuming that `ns` contains `n` as well as all the local endpoints. Is that safe?